packages/expo-cli/src/commands/run/buildAndroidClientAsync.ts

```diff
@@ -0,0 +1,50 @@
+import { AndroidConfig } from '@expo/config-plugins';
+import spawnAsync from '@expo/spawn-async';
+import { Android } from '@expo/xdl';
+import ora from 'ora';
+import path from 'path';
+
+type Options = {
+  buildVariant: string;
+};
+
+function getGradleTask(buildVariant: string): string {
+  return `install${buildVariant.charAt(0).toUpperCase()}${buildVariant.slice(1)}`;
+}
+
+export default async function buildAndroidClientAsync(
+  projectRoot: string,
+  options: Options
+): Promise<void> {
+  const devices = await Android.getAllAvailableDevicesAsync();
+  const device = devices.length > 1 ? await Android.promptForDeviceAsync(devices) : devices[0];
+  if (!device) {
+    return;
+  }
+  const bootedDevice = await Android.attemptToStartEmulatorOrAssertAsync(device);
+  if (!bootedDevice) {
+    return;
+  }
+
+  const spinner = ora('Building app ').start();
```

Comment:
build logs?

Reply:
Looks like this interferes with them, so I'm going to remove the spinner and use `Log.log` instead.